code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -20,9 +20,50 @@
 	drop_sound = 'sound/items/drop/axe.ogg'
 	pickup_sound = 'sound/items/pickup/axe.ogg'
 
+//crafting / welding vars
+	var/datum/material/material //*sigh* i guess this is how we're doing this.

```

Comment:
You _could_ make the tiles a subtype of stack/material, but that almost seems like more work...
Someday, we'll have material on /obj/item (Or maybe even /obj???)

Reply:
(Doesn't need a change)

Reply:
considered it but then also i realised i value my sanity